tests/scripts/task_config_build_cpu.sh

```diff
@@ -54,7 +54,7 @@ echo set\(USE_ETHOSN_HW OFF\) >> config.cmake
 echo set\(USE_CMSISNN OFF\) >> config.cmake
 echo set\(USE_VITIS_AI ON\) >> config.cmake
 echo set\(USE_VERILATOR ON\) >> config.cmake
-echo set\(USE_LIBBACKTRACE ON\) >> config.cmake
+echo set\(USE_LIBBACKTRACE COMPILE\) >> config.cmake
```

Comment:
Could you please update the comments in `config.cmake`?

Reply:
It's updated. Also removed the comments from Logging.cmake to avoid duplication.